### PR TITLE
chore: remove headed setting

### PR DESCRIPTION
**Pull request by pavelfeldman**

### CODE REVIEW

**Review by mxschmitt — APPROVED** [review]

File: src/reusedBrowser.ts [hunk 1]
```diff
@@ -169,6 +179,8 @@ test('test', async ({ page }) => {
   }
 
   async willRunTests(config: TestConfig) {
+    if (!this._checkVersion(config, 'Show & reuse browser'))
```
Comment (dgozman) [inline src/reusedBrowser.ts:182]: Why is this here, and not in `startIfNeeded`? Do we only get here if "show & reuse" is checked?

**Review by dgozman — APPROVED** [review]
